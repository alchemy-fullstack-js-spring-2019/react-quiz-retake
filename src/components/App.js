import React from 'react';
import Meme from '../containers/Meme';

export default function App() {
  return (
    <>
    <h1>Make yer Meme</h1>
    <Meme />
    </>
  );
}
