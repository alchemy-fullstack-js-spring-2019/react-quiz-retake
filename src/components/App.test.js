import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('meme display component', () => {
  it('renders a meme display', () => {
    const wrapper = shallow(<App headerText="hi" imgSource="blue" footerText="test" />);
    expect(wrapper).toMatchSnapshot();
  });
});