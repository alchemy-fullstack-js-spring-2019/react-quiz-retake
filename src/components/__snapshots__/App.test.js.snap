// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`meme display component renders a meme display 1`] = `<Meme />`;
